lib/puppet/type/group.rb

```diff
@@ -99,7 +99,12 @@ def insync?(current)
       def is_to_s(currentvalue)
         if provider.respond_to?(:members_to_s)
           currentvalue = '' if currentvalue.nil?
-          return provider.members_to_s(currentvalue.split(','))
+          currentvalue = currentvalue.is_a?(Array) ? currentvalue : currentvalue.split(',')
+
+          begin
+            return provider.members_to_s(currentvalue)
+          rescue Puppet::Util::Windows::Error
+          end
```

Comment:
Does it fail specifically by having `Puppet::Util::Windows::Error`? I'm guessing not as that file has been loaded and all is well.